qiskit_nature/results/utils/protein_shape_file_gen.py

```diff
@@ -0,0 +1,156 @@
+        )
+        self._side_chain_aminoacid_list = np.array(
+            [
+                aminoacid.residue_sequence[0] if (aminoacid is not None) else None
+                for aminoacid in peptide.get_side_chains()
+            ]
+        )
+        self._main_positions = self.generate_main_positions()
+        self._side_positions = self.generate_side_positions()
+
+    def generate_side_positions(self) -> List[Optional[np.ndarray]]:
+        """Generates the positions of the side chain."""
+        side_positions: List[Optional[np.ndarray]] = []
+        counter = 1
+        for mainpos, sideturn in zip(self.main_positions, self._side_chain_turns):
+            if sideturn is None:
+                side_positions.append(None)
+            else:
+                side_positions.append(mainpos + (-1) ** counter * self.COORDINATES[sideturn])
+
+            counter += 1
+        return side_positions
+
+    @property
+    def side_positions(self) -> List[Optional[np.ndarray]]:
+        """
+        Returns the xyz position for each side chain element.
+
+        Returns:
+            A list with the position of the side chain of each bead in the main chain in order.
+            None in the i-th position of the list corresponds to no side chain at that
+            position of the main chain.
+
+        """
+
+        return self._side_positions
+
+    def generate_main_positions(self) -> np.ndarray:
+        """Generates the positions of the main chain."""
+        length_turns = len(self._main_chain_turns)
+        relative_positions = np.zeros((length_turns + 1, 3), dtype=float)
+
+        for i in range(length_turns):
+            relative_positions[i + 1] = (-1) ** i * self.COORDINATES[self._main_chain_turns[i]]
+
+        return relative_positions.cumsum(axis=0)
+
+    @property
+    def main_positions(self) -> np.ndarray:
+        """
+        Returns an array with the cartesian coordinates of each aminoacid in the main chain.
+        The first time called it generates the coordinates.
+
+        Returns:
+            An array with the cartesian coordinates of each aminoacid in the main chain.
+
+        """
+
+        return self._main_positions
+
+    def save_xyz_file(self, name: str, path: str = "", comment: str = "") -> None:
+        """
+        Saves the data as an .xyz file.
+        For more information about .xyz files see:
+        https://en.wikipedia.org/wiki/XYZ_file_format.
+
+        Args:
+            name: The file will be called name `.xyz`. Can overwrite files.
+            path: Path under which the file will be saved. If no path is specified the file will
+                be saved in the current working directory.
+            comment: Comment to be added to the second line of the file. By default the line will
+                be left blank.
+        """
+        data = self.get_xyz_file()
+        number_of_particles = data.shape[0]
+        header = f"{number_of_particles}\n#{comment}"
```

Comment:
```suggestion
        header = f"{number_of_particles}\n{comment}"
```
There is no need for this `#` symbol